predict.py

```diff
@@ -0,0 +1,302 @@
+import sys
+import os
+import tempfile
+import warnings
+from pathlib import Path
+import nltk
+import torch
+from torch import nn
+import torchvision.transforms as transforms
+import numpy as np
+import imageio
+from PIL import Image as Image_PIL
+from scipy.stats import truncnorm
+from nltk.corpus import wordnet as wn
+import cma
+import sklearn.metrics
+import cog
+
+sys.path.insert(0, "stylegan2_ada_pytorch")
```

Comment:
Not using stylegan models so I think this line is unnecessary.

Reply:
Thank you for all the comments, I have updated the others accordingly. 
However this line seems necessary, I added it to avoid this error: As `import inference.utils as inference_utils` is  in `predict.py`, and `inference.utils` has this import `import stylegan2_ada_pytorch.legacy as legacy`,  and in `stylegan2_ada_pytorch/legacy.py` it has `import dnnlib`. without that line 19 in `predict.py` it will complain not being able to find `dnnlib`.